align/pool2d/gen_tensors.sh

```diff
@@ -0,0 +1,9 @@
+#! /usr/bin/env bash
+
+eval "$(conda shell.bash hook)";
```

Comment:
What type of dependency on the user using conda does this create? Does this only impact CI?

Reply:
I think we have scripts like this (see for example, [here](https://github.com/flexflow/FlexFlow/blob/master/align/linear/gen_tensors.sh)) because we need to run the FlexFlow side of the alignment computations in a environment with CPU-only Pytorch, but then we will need GPU-based Pytorch if we want to run the pytorch computations on GPU. Idk if there is a way to generalize the script to allow it to also turn on/off non-conda environments? If not, I think supporting only conda is fine for the test